packages/test-helpers/src/enzyme.ts

```diff
@@ -14,16 +15,37 @@ export const axeDefaultOptions: AxeOptions = {
   },
 }
 
+export interface MountOptions {
+  mountOptions?: MountRendererProps
+  axeOptions?: AxeOptions
+  act?: 'sync' | 'async'
+}
+
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export const mountAndCheckA11Y = async <P>(
```

Comment:
@aigoncharov Maybe it'd help if we add some documentation here? I've never used your helpers before and I have to study the code in order to understand what it does.

Reply:
@jvorcak do you think it would be OK to address it later, once we have released the first draft of the `ui`? I have created a separate task for it
https://github.com/hazelcast/frontend-shared/issues/51

Reply:
@aigoncharov sure!